Documentation/Signoffs/Sensor_Signoff.md

```diff
@@ -0,0 +1,86 @@
+
+6:  Each sensor will contain wiring to communicate with the DAQ. This wiring must meet the codes listed to ensure safety and correct operation.
+
+16:  This constraint is set and satisfied by the DAQ subsystem, which has ports that can supply or receive 0-10 volts.
+
+19:  Due to the electric motorcycle’s data sheets, it has been calculated that the test data must be able to be captured in these ranges:
+
+RPM:  0-1,288 RPM
+
+Torque:  0-43Nm (calculated using 31.13 lb-ft with a conversion ratio of 1lb-ft = 1.3558 Nm)
+
+## Schematics
+
+### Rotary Torque Sensor Application:
+
+ ![image](https://github.com/user-attachments/assets/d9eba0f2-4f64-4a90-9811-57e863a3841e)
+
+
+The image above shows where the rotary torque sensor shall be installed within the dynamometer's mechanical system. It shall be installed between the drum (labeled Load Equipment) and one of two electric motors (labeled Power Equipment). The sensor shaft has a coupling on each side that connects the solid protruding shaft of the sensor. Then, the sensor can measure the RPM seen by an internal encoder and the torque applied by an integral strain gauge attached to the solid protruding shaft.
+
+
+### Rotary Torque Sensor Wiring Schedule:
+ ![image](https://github.com/user-attachments/assets/cc51497b-23e1-4309-8449-b6eeedec3bc1)
+ ![image](https://github.com/user-attachments/assets/f9c313e3-b487-4814-b57c-716658497eb2)
+
+The image above shows the wiring schedule for the torque sensor cable. The connections are to the DAQ (NI USB_6215) and a USB to RS485 converter. This allows the user to program the sensor's parameters and provides a redundant data display to ensure the DAQ output matches the sensor's output. The ninth pin is connected to the cable shield, which will be attached to terminal 11 on the DAQ to eliminate any chance of noise generated onto the signal from electromagnetic interference.
+
+
+
+### Rotary Torque Sensor Installation Dimensions:
+
+![image](https://github.com/user-attachments/assets/04b262e3-b051-4b7a-8d39-33ea47275d50)
+
+The image above shows the required dimensions in millimeters for mounting the sensor into the dynamometer.
+
+## Analysis
+### Rotary Torque Sensor:
+The torque sensor is a transducer that converts mechanical torque into an electric output signal. This is achieved by attaching strain gauges to the shaft. As the shaft's torsion increases, the strain gauges' value proportionally changes, and the change in voltage or resistance measured is used to calculate the shaft's induced torque. The bike’s maximum torque from the 3500 W drive motor is 25.94 lb-ft. With a 20% safety factor, the system must then be able to measure up to 31.13 lb-ft. The selected sensor can support up to 100 Nm, which converts to 73.756 lb-ft. 
```

Comment:
Is 25.94 lb-ft the bike or the drum max torque? Remember we are reading the drum.

Reply:
The 25.95 lb-ft, which was re-calculated to be 19.13 lb-ft, is the drum's max torque which is what we're measuring with the sensor.